gem/tests/test_views.py

```diff
@@ -369,6 +378,24 @@ def test_comment_shows_user_display_name(self):
         self.assertContains(response, "this is my alias")
         self.assertNotContains(response, "tester")
 
+    def test_anonymous_comment_translation(self):
+        self.user.profile.alias = 'this is my alias'
+        MoloComment.objects.create(
+            content_object=self.translated_article,
+            object_pk=self.translated_article.id,
+            content_type=ContentType.objects.get_for_model(
+                self.translated_article),
+            site=Site.objects.get_current(), user=self.user,
+            comment='This is another comment for French',
+            submit_date=timezone.now())
+        response = self.client.get('/locale/fr/')
+        response = self.client.get(
+            reverse('molo.commenting:more-comments', args=(
+                self.translated_article.pk,)))
+        self.assertContains(response, "This is another comment for French")
+        self.assertContains(response, "Anonyme")
```

Comment:
small typo here, I think you need to check for `Anonymous`

Reply:
this is the french article, it should be "Anonyme"